src/test/java/nextstep/subway/station/StationAcceptanceTest.java

```diff
@@ -111,5 +121,50 @@ void getStations() {
     @DisplayName("지하철역을 제거한다.")
     @Test
     void deleteStation() {
+//        given
+        이름으로_지하철역을_생성("강남역");
+
+//        when
+        ExtractableResponse<Response> response = ID값으로_지하철역_삭제(1L);
+
+//        then
+        응답결과로_NO_CONCENT를_응답(response);
+    }
+
+    private ExtractableResponse<Response> ID값으로_지하철역_삭제(Long id) {
+        return RestAssured.given().log().all()
+                .when().delete("/stations/" + id)
+                .then().log().all()
+                .extract();
+    }
+
+    private AbstractIntegerAssert<?> 응답결과로_NO_CONCENT를_응답(ExtractableResponse<Response> response) {
+        return assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+    }
+
+    private void 두개의_지하철역_응답을_받음(ExtractableResponse<Response> response) {
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertThat(response.jsonPath().getList(".").size()).isEqualTo(2);
+    }
+
+    private void 이름으로_지하철역을_생성(String name) {
```

Comment:
```suggestion
    private void 지하철역을_생성(String name) {
```
파라미터를 통해 알 수있는 정보라고 생각하는데 승인님은 어떻게 생각하시나요?

Reply:
충분히 파라미터를 통해 유추할 수 있는 정보 `지하철역을_생성`으로 하는것이 깔끔한것 같네요.
추후에 진행하는 미션에서 네이밍을 할 때 신경써보겠습니다 :)